apps/dotcom/client/src/tla/styles/tla.css

```diff
@@ -98,6 +98,10 @@
 		inset 0px 0px 0px 1px var(--tla-color-panel);
 }
 
+.tla-focus-mode .tl-watermark_SEE-LICENSE {
+	visibility: hidden;
```

Comment:
i'll let Steve weigh in on whether we should do this or not, but little nit here would be that `display: none` would feel slightly more correct.